### PR TITLE
LoadPanel: set propagateOutsideClick: true by default

### PR DESCRIPTION
**Pull request by dayanch-n**

### CODE REVIEW

File: js/ui/load_panel.js [hunk 1]
```diff
@@ -54,28 +54,9 @@ const LoadPanel = Overlay.inherit({
...
-            * @hidden
-            */
-            resizeEnabled: false,
-
-            focusStateEnabled: false
-
-            /**
-            * @name dxLoadPanelOptions.dragEnabled
-            * @hidden
-            */
```
Comment (ksercs) [inline js/ui/load_panel.js]: Why did u remove this default values and doccoments? This changes seem to be excess

**Review by ksercs — CHANGES_REQUESTED** [review]

File: js/ui/load_panel.js (hunk 1 above)
Comment (dayanch-n) [reply js/ui/load_panel.js]: I have compared with upstream 21_2 and those were not there. So I thought it was outdated.

Comment (dayanch-n) [reply js/ui/load_panel.js]: Should I add them back?

Comment (Zedwag) [reply js/ui/load_panel.js]: https://github.com/DevExpress/DevExtreme/blob/21_2/js/ui/load_panel.js#L66 it exsist in 21_2

**Review by Zedwag — APPROVED** [review]

**Review by ksercs — APPROVED** [review]

**Pull request merged**